SourceryRuntime/Sources/AST/Type.swift

```diff
@@ -440,3 +464,9 @@ extension Type {
         return !isNotClass && !isExtension
     }
 }
+
+private extension Sequence where Element == Import {
+    func removingDuplicates() -> [Import] {
+        return Array(Set(self))
```

Comment:
imagine someone writes {{ type.allImports }} in the template. Every time it can yield different result resulting in unneeded changes because Set will loose the ordering

Reply:
fixed